### DISCUSSION TITLE
Dynamic retrieval of domain/unique values of queryables

### DISCUSSION THREAD

**Discussion by schmity** [ldproxy/ldproxy#1128] [Setup]
Looking for a way to publish domain/unique values for queryables with ldproxy, I came across this issue in the OGC API - Features Repo: https://github.com/opengeospatial/ogcapi-features/issues/799#issuecomment-1458367695, which suggests to list the values as enum in the queryables schema.
Is there an option in ldproxy to retrieve the domain/unique values from the database dynamically?

**Comment by cportele** [comment]
Currently not, at the moment the values are set in the feature provider schema. Right now, the list of values is usually taken from a data specification/model, independent of the range of values that exists in the data. But the values could also be derived from the data in an SQL feature provider.
